api/src/main/java/com/exelaration/abstractmemery/services/implementations/ImageServiceImpl.java

```diff
@@ -31,9 +31,13 @@ public Image save(MultipartFile file) throws IOException {
     } catch (Exception e) {
       e.printStackTrace();
     }
+    try {
+      fileStorageService.save(fileName, file.getInputStream());
+      return metadataService.save(image);
 
-    fileStorageService.save(fileName, file.getInputStream());
-
-    return metadataService.save(image);
+    } catch (Exception e) {
+      Image nullImage = new Image();
```

Comment:
This isn't a null `Image`, this is a default `Image` object.  I think this should actually just return `null`.